code/defines/procs/AStar.dm

```diff
@@ -37,147 +37,152 @@ length to avoid portals or something i guess?? Not that they're counted right no
 // Also added 'exclude' turf to avoid travelling over; defaults to null
 
 
-PriorityQueue
+/PriorityQueue
 	var/L[]
 	var/cmp
-	New(compare)
-		L = new()
-		cmp = compare
-	proc
-		IsEmpty()
-			return !L.len
-		Enqueue(d)
-			var/i
-			var/j
-			L.Add(d)
-			i = L.len
-			j = i>>1
-			while(i > 1 &&  call(cmp)(L[j],L[i]) > 0)
-				L.Swap(i,j)
-				i = j
-				j >>= 1
-
-		Dequeue()
-			if(!L.len) return 0
-			. = L[1]
-			Remove(1)
-
-		Remove(i)
-			if(i > L.len) return 0
-			L.Swap(i,L.len)
-			L.Cut(L.len)
-			if(i < L.len)
-				_Fix(i)
+
+/PriorityQueue/New(compare)
+	L = new()
+	cmp = compare
+
+/PriorityQueue/proc/IsEmpty()
+	return !L.len
+
+/PriorityQueue/proc/Enqueue(d)
+	var/i
+	var/j
+	L.Add(d)
+	i = L.len
+	j = i>>1
+	while(i > 1 &&  call(cmp)(L[j],L[i]) > 0)
+		L.Swap(i,j)
+		i = j
+		j >>= 1
+
+/PriorityQueue/proc/Dequeue()
+	if(!L.len) return 0
+	. = L[1]
+	Remove(1)
+
+/PriorityQueue/proc/Remove(i)
+	if(i > L.len) return 0
+	L.Swap(i,L.len)
+	L.Cut(L.len)
+	if(i < L.len)
 		_Fix(i)
-			var/child = i + i
-			var/item = L[i]
-			while(child <= L.len)
-				if(child + 1 <= L.len && call(cmp)(L[child],L[child + 1]) > 0)
-					child++
-				if(call(cmp)(item,L[child]) > 0)
-					L[i] = L[child]
-					i = child
-				else
-					break
-				child = i + i
-			L[i] = item
-		List()
-			var/ret[] = new()
-			var/copy = L.Copy()
-			while(!IsEmpty())
-				ret.Add(Dequeue())
-			L = copy
-			return ret
-		RemoveItem(i)
-			var/ind = L.Find(i)
-			if(ind)
-				Remove(ind)
-PathNode
+
+/PriorityQueue/proc/_Fix(i)
+	var/child = i + i
+	var/item = L[i]
+	while(child <= L.len)
+		if(child + 1 <= L.len && call(cmp)(L[child],L[child + 1]) > 0)
+			child++
+		if(call(cmp)(item,L[child]) > 0)
+			L[i] = L[child]
+			i = child
+		else
+			break
+		child = i + i
+	L[i] = item
+
+/PriorityQueue/proc/List()
+	var/ret[] = new()
+	var/copy = L.Copy()
+	while(!IsEmpty())
+		ret.Add(Dequeue())
+	L = copy
+	return ret
+
+/PriorityQueue/proc/RemoveItem(i)
+	var/ind = L.Find(i)
+	if(ind)
+		Remove(ind)
+/PathNode
 	var/datum/source
 	var/PathNode/prevNode
 	var/f
```

Comment:
Do we need to have lots of single character variables here? If this is all consistent with upstream then sure but it's cringe

Reply:
deeply, i’m not sure if tg even has